README.md

```diff
@@ -54,6 +54,8 @@ reproduce the problem and attach the collected logs to the JIRA issue.
 ## Changelog
 
 ### Version 1.38 (2019-10-17)
+-   **Attention: Needs Java 8 at least. If you're still using Java 7,
```

Comment:
I am not sure if such a disclaimer is required, as I'd expect people to be not able to install this version on a Jenkins version that still runs on Java 7.

Reply:
Even better. I just updated the commit and remove that Java 7 traces.